enterprise/backend/test/metabase_enterprise/serialization/v2/e2e_test.clj

```diff
@@ -654,19 +663,25 @@
                 (is (serdes/with-cache (serdes.load/load-metabase! (ingest/ingest-yaml dump-dir)))
                     "successful"))
               (testing "Series are loaded correctly"
-                (let [new-dc1-id (t2/select-one-pk :model/DashboardCard :card_id (:id c1))
-                      new-dc2-id (t2/select-one-pk :model/DashboardCard :card_id (:id c2))
```

Comment:
@piranha The problem was that these two lines use the `id` of `c1` and `c2`. But serialization doesn't guarantee stable ids.

Reply:
Ahhh I see!